docker-compose-localstack-tests.yml

```diff
@@ -15,7 +15,6 @@ services:
       - DOCKER_HOST=unix:///var/run/docker.sock
       - DEFAULT_REGION=eu-west-2
     volumes:
-      - "${TMPDIR:-/tmp/localstack}:/tmp/localstack"
```

Comment:
Not sure we need this; seems to cause issues after a host goes to sleep - not sure why, but treat this as an experiment